build.gradle

```diff
@@ -5,7 +5,7 @@ subprojects {
     apply plugin: 'maven-publish'
 
 
-    version = "2.3.0"
+    version = "2.4.0"
```

Comment:
protected API changes are major because they are not backwards compatible

Reply:
Updated to 3.0.0